src/gui/plugins/view_angle/ViewAngle.cc

```diff
@@ -311,6 +327,82 @@ void ViewAngle::SetCamPose(double _x, double _y, double _z,
   }
 }
 
+/////////////////////////////////////////////////
+bool ViewAngle::OnMoveToModelService(const ignition::msgs::GUICamera &_msg,
+  ignition::msgs::Boolean &_res)
+{
+  std::lock_guard<std::mutex> lock(this->dataPtr->mutex);
+  auto scene = this->dataPtr->camera->Scene();
+
+  auto visualToMove = scene->VisualByName(_msg.name());
+  if (nullptr == visualToMove)
+  {
+    ignerr << "Failed to get visual with ID ["
+           << _msg.name() << "]" << std::endl;
+    _res.set_data(false);
+    return false;
+  }
+  Entity entityId = kNullEntity;
+  try
+  {
+    entityId = std::get<int>(visualToMove->UserData("gazebo-entity"));
```

Comment:
we need to make sure to change this from `int` to `unsigned int` when merging forward to gz-sim7